Afanasenko_Oleksii/HW4_task2_afanasenko_oleksii.py

```diff
@@ -0,0 +1,8 @@
+list_1 = [2]
+result_1 = []
+for number in list_1:
+    if number * 2 == 4:
+        result_1.append((number * number))
```

Comment:
why do you use a loop for this? 
I asked just to * and / in two different ways. 
First one: standard math way, second using byte operations